pytorch/training/docker/2.2/py3/Dockerfile.cpu

```diff
@@ -250,10 +250,6 @@ ARG PYTHON
 ARG PYTORCH_VERSION
 ARG CONDA_CHANNEL
 
-# Selectively enable PyTorch Verbose Logging
```

Comment:
This is removed due to this [thread](https://amzn-aws.slack.com/archives/C06EZT2RB0B/p1707771786863799)